scripts/ci/_utils.sh

```diff
@@ -1653,6 +1654,27 @@ function run_prepare_packages() {
         | tee -a "${OUTPUT_LOG}"
 }
 
+# Docker command to generate release notes for backport packages
+function run_generate_readme() {
```

Comment:
This is only used from `scripts/ci/ci_generate_backport_readme.sh` right? If so I'd like it to live there, otherwise it makes it much harder to see what each script does when the bulk of it's job is calling a (non-reused) function in another file.

Reply:
Indeed. Good point. will fix it.

Reply:
Actually. I realixed why I did it - I forgot to add breeze command for this (and prepare packages). This is now fixed - breeze has now two more nice commands and the function is in the _utils.sh so that it can be used in both.